pkg/virt-api/rest/subresource.go

```diff
@@ -288,6 +288,18 @@ func (app *SubresourceAPIApp) StartVMRequestHandler(request *restful.Request, re
 		return
 	}
 
+	vmi, err := app.VirtCli.VirtualMachineInstance(namespace).Get(name, &k8smetav1.GetOptions{})
```

Comment:
@stu-gott Already saw that there's a method `SubresourceAPIApp.fetchVirtualMachineInstance` for this, will refactor.